SampleApp/SampleApp.Android/Assets/airshipconfig.properties

```diff
@@ -1,7 +1,7 @@
-developmentAppKey = Your Development App Key
-developmentAppSecret = Your Development App Secret
-productionAppKey = Your Production App Key
-productionAppSecret = Your Production Secret
+developmentAppKey = Your development app key
+developmentAppSecret = Yopur development app secret
+productionAppKey = Your production app key
+productionAppSecret = Your production app secret
 
 gcmSender = Your GCM sender ID is your Google API project number (required for GCM)
```

Comment:
should be removed

Reply:
Done ✅

Reply:
I'm pretty sure @rlepinski wanted you to remove just the `gcmSender` line.

Reply:
(I added it back)